junwoo/controllers/postController.js

```diff
@@ -0,0 +1,52 @@
+//controller/postController.js
+
+const postService = require("../services/postService");
+
+const postUp = async (req, res) => {
```

Comment:
함수명 수정해주세요.
함수명은 어떤 행위와 결과가 예측될 수 있도록 동사+목적어 등이 포함되도록 작성해주는 것이 좋습니다.

```suggestion
const createPost = async (req, res) => {
```